lib/components/term.js

```diff
@@ -98,9 +107,43 @@ export default class Term extends Component {
 
     this.getScreenNode().addEventListener('mouseup', this.handleMouseUp);
     this.getScreenNode().addEventListener('mousedown', this.handleMouseDown);
+    this.getScreenNode().addEventListener('contextmenu', this.handleRightClick);
     terms[this.props.uid] = this;
   }
 
+  handleRightClick(e) {
+    e.preventDefault();
+    e.stopPropagation();
+
+    const selection = e.view.getSelection();
+    const {isCollapsed} = selection;
+    const selectedText = selection.toString();
+
+    const selectionMenu = [
+      {
+        type: 'separator'
+      },
+      {
+        label: 'Open Selection as URL',
+        click() {
+          openExternal(selectedText);
+        },
+        enabled: !isCollapsed
+      },
+      {
+        label: 'Search the Web for Selection',
```

Comment:
in https://github.com/zeit/hyper/pull/2001/commits/b3a9d307f1c55b38e7267acaa773460953dd8c1e I've made this configurable via exposing a new key/value (`searchEngineUrl`) from config so this can be customized via `.hyper.js`

defaults to google, but can be anything the user desires.

this would likely need to be address in `app/config/config-default.js` as well (similar to `bellSoundURL`)

/cc @henrikdahl @CodeTheory